src/v/kafka/protocol/protocol.bzl

```diff
@@ -0,0 +1,22 @@
+"""
+This module contains functions for working with Kafka messages.
+"""
+
+load("//bazel:build.bzl", "redpanda_cc_library")
+load("//src/v/kafka/protocol/schemata:generator.bzl", "MESSAGES")
+
+def generate_kafka_messages(name = "generate_kafka_messages"):
```

Comment:
maybe something like `generate_kafka_message_libs`

Reply:
fixed in latest push